cli/azd/docs/md/azd.md

```diff
@@ -0,0 +1,847 @@
+---
+title: Azure Developer CLI Preview reference
```

Comment:
Just checking -- are you intending to include adding `azd.md` in this change as well?

Reply:
I did not!  Good catch.